### PR TITLE
docs: improve README-zh-CN.md for GitHub and Gitlab

### PR DESCRIPTION
**Pull request by mindlesscloud**
# Summary
improve `README-zh-CN.md` for GitHub and Gitlab

### Key Points

- [ ] This is a breaking change
- [ ] New or existing documentation is updated

### Description
Describe what this PR does, and aims to solve in a few sentences.

### Does this close any open issues?
https://github.com/merico-dev/lake/issues/779

### Current Behavior
Describe the current behaviour of this issue, if relevant.

### New Behavior
Describe the new behaviour updated in this issue, if relevant.

### Screenshots
Include any relevant screenshots here.

### Other Information
Any other information that is important to this PR.

### CODE REVIEW

File: plugins/github/README-zh-CN.md [hunk 1]
```diff
@@ -36,22 +36,28 @@ https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/c
 
 ## 配置
 
-在`.evn`文件中需要设置如下配置项
+### 数据源连接配置
+配置界面需要填入以下字段
+- **Connection Name** [`只读`]
+    - ⚠️ 默认值为 "**Github**" 请不要改动。
+- **Endpoint URL** (REST URL, 以 `https://`或`http://`开头)
+    - 应当填入可用的REST API Endpoint。例如 `https://api.github.com/`
```
Comment (Startrekzky) [inline plugins/github/README-zh-CN.md:44]: 这个地方重点说明一下需要加 slash ？

File: plugins/gitlab/README-zh-CN.md [hunk 2]
```diff
@@ -19,26 +18,42 @@
 
 ## 配置
 
-插件运行前，你需要在 `.env` 文件中进行以下配置：
+### 数据源连接配置
+配置界面需要填入以下字段
+- **Connection Name** [`只读`]
+    - ⚠️ 默认值为 "**Gitlab**" 请不要改动。
+- **Endpoint URL** (REST URL, 以 `https://`或`http://`开头)
+    - 应当填入可用的REST API Endpoint。例如 `https://gitlab.example.com/api/v4/`
```
Comment (Startrekzky) [inline plugins/gitlab/README-zh-CN.md:26]: 这里也得加 slash 吗？

**Review by Startrekzky — CHANGES_REQUESTED** [review]

File: plugins/github/README-zh-CN.md (hunk 1 above)
Comment (mindlesscloud) [reply plugins/github/README-zh-CN.md:44]: ok

File: plugins/gitlab/README-zh-CN.md (hunk 2 above)
Comment (mindlesscloud) [reply plugins/gitlab/README-zh-CN.md:26]: 是的

**Review by klesh — APPROVED** [review]
LGTM

**Pull request merged**